python/modules/Stop0lBaselineProducer.py

```diff
@@ -64,6 +64,8 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
         self.out.branch("Pass_HEMVeto30"     + self.suffix, "O", title="HEM Veto 2018: eta[-3, -1.4], phi[-1.57, -0.87], pt > 30")
         self.out.branch("Pass_exHEMVeto20"   + self.suffix, "O", title="HEM Veto 2018: eta[-3.2, -1.2], phi[-1.77, -0.67], pt > 20")
         self.out.branch("Pass_exHEMVeto30"   + self.suffix, "O", title="HEM Veto 2018: eta[-3.2, -1.2], phi[-1.77, -0.67], pt > 30")
+	self.out.branch("Stop0l_nJetIdx"     + self.suffix, "I")
+	self.out.branch("Stop0l_JetIdx"      + self.suffix, "I", lenVar="Stop0l_nJetIdx")
```

Comment:
Let us continue to use Jet_sortedIdx or some thing similar. As I would consider this is a property of Jet, not Stop0l.

Reply:
@mkilpatr Can you follow up this comment?

Reply:
Yes I’ll change it.